pkg/roachprod/roachprod.go

```diff
@@ -1462,6 +1462,7 @@ func Create(
 		// No need for ssh for local clusters.
 		return LoadClusters()
 	}
+	l.Printf("Created cluster %s; setting up SSH...", clusterName)
```

Comment:
Did you intend to commit these changes in this PR? Just making sure as they are not GC related.

Reply:
During testing, `roachprod create` failed several times in Azure during `SetupSSH`. I wasn't able to reproduce it, so chalking it up to a transient error. However, the above log message was useful for debugging since we can better isolate the failed step.